setup.py

```diff
@@ -12,22 +12,89 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-import setuptools
 from setuptools import setup
+from setuptools.command.build_ext import build_ext
+from setuptools.extension import Extension
+
 import versioneer
 
 # read the contents of your README file
 from pathlib import Path
 this_directory = Path(__file__).parent
 long_description = (this_directory / "README.md").read_text()
 
+ZITI_SDK_BASE = 'https://github.com/openziti/ziti-sdk-c/releases/download'
+
+
+class GetZitilib(build_ext):
+
+    def build_extension(self, ext) -> None:
+        import platform
```

Comment:
You don't really save any time by importing in the functions here. These should all be at the top of the module per styling guidelines.